wolox-books/src/components/BookInfo.vue

```diff
@@ -0,0 +1,75 @@
+<template lang='pug'>
+.book-container.row.back-grey
+  img.book-image.margin-20(:src='book.image_url' :alt='book.title')
+  .column.width-100
+    .row.pad-20.under-green-line
+      span.bold.text-ending.pad-right-5
+        | {{book.title}}
+      span.text-ending.pad-right-5
+        | ({{book.genre}})
+    .border
+      .row.pad-5
+        span.text-ending.bold
+          | Book Author:
+        span.text-ending
+          | {{book.author}}
+      .row.pad-5
+        span.text-ending.bold
+          | Publisher:
+        span.text-ending
+          | {{book.publisher}}
+      .row.pad-5
+        span.text-ending.bold
+          | Year of publication:
+        span.text-ending
+          | {{book.year}}
+</template>
+
+<script>
+import { getBookDetails } from '@/services/bookService'
+
+export default {
+  data() {
+    return {
+      bookId: this.$route.params.id,
```

Comment:
You already got this in `this.$route.params.id`, why copying it again in the data? :thinking: